admin/class-admin-asset-manager.php

```diff
@@ -20,17 +20,24 @@ class WPSEO_Admin_Asset_Manager {
 	 */
 	const PREFIX = 'yoast-seo-';
 
+	/**
+	 * @var string Public accessible prefix for naming the assets.
+	 */
+	public $prefix;
```

Comment:
I would made this property protected, because there is no need to use it outside this class.

Reply:
Actually I think there is. We will be using it in Local SEO.
Example:

```
$asset_manager = new WPSEO_Admin_Asset_Manager( new WPSEO_Admin_Asset_SEO_Location( WPSEO_LOCAL_FILE ), 'wp-seo-local-');
$asset_manager->register_scripts( array( array(
	'name' => 'frontend',
	'src' => $asset_manager->prefix . 'frontend-' . $asset_manager->flatten_version( WPSEO_LOCAL_VERSION ),
	'version' => WPSEO_LOCAL_VERSION,
	'deps' => array( 'jquery' ),
) ) );

$asset_manager->enqueue_script( 'frontend' );

wp_localize_script( $asset_manager->prefix . 'frontend', 'wpseo_local_data', array(
	'ajaxurl'                   => 'admin-ajax.php',
	'adminurl'                  => admin_url(),
	'has_multiple_locations'    => wpseo_has_multiple_locations(),
	'unit_system'               => ! empty( $options['unit_system'] ) ? $options['unit_system'] : 'METRIC',
	'default_country'           => $default_country,
	'detect_location'           => $detect_location,
	'marker_cluster_image_path' => apply_filters( 'wpseo_local_marker_cluster_image_path', esc_url( trailingslashit( plugin_dir_url( dirname( __FILE__ ) ) ) . 'images/m' ) ),
) );
```

Reply:
Alright, new idea. I'll handle that in the Local SEO plugin itself.